com.unity.mobile.notifications/Runtime/Android/AndroidNotificationCenter.cs

```diff
@@ -212,7 +212,10 @@ public static int SendNotification(AndroidNotification notification, string chan
             if (!Initialize())
                 return -1;
 
-            int id = Math.Abs(DateTime.Now.ToString("yyMMddHHmmssffffff").GetHashCode()) + (new System.Random().Next(10000));
+            // Now.ToString("yyMMddHHmmssffffff"), but avoiding any culture-related formatting or dependencies
+            var now = DateTime.UtcNow;
+            var nowFormatted = $"{now.Year}{now.Month}{now.Day}{now.Hour}{now.Minute}{now.Second}{now.Millisecond}";
+            int id = Math.Abs(nowFormatted.GetHashCode()) + (new System.Random().Next(10000));
```

Comment:
Wondering if this new System.Random().Next(10000)) won't cause a situation where id in ultra rare cases will be the same